pallets/pallet-did-lookup/src/lib.rs

```diff
@@ -421,6 +431,19 @@ pub mod pallet {
 				Err(Error::<T>::NotFound.into())
 			}
 		}
+
+		#[cfg(any(feature = "try-runtime", test))]
+		pub fn do_try_state() -> DispatchResult {
+			ConnectedDids::<T>::iter().try_for_each(|(account, record)| -> DispatchResult {
+				let is_in_connected_accounts = ConnectedAccounts::<T>::contains_key(record.did, account);
+
+				ensure!(is_in_connected_accounts, DispatchError::Other("Test"));
+
+				Ok(())
+			})?;
```

Comment:
We can also simply return the result of the `try_for_each` without writing the additional `Ok(())` line at the end.